packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/utils.tsx

```diff
@@ -117,4 +117,7 @@ export function findServerChildSpan(spans: Span[]) {
   return null;
 }
 
+export const isKindClient = (span: Span) =>
```

Comment:
```suggestion
export function isKindClient(span: Span): Boolean =>
```

Reply:
I think this should be  
```
export const isKindClient = (span: Span):Boolean =>
  span.tags.some(({ key, value }) => key === 'span.kind' && value === 'client');
```
or
```
export function isKindClient(span: Span):Boolean {
  return span.tags.some(({ key, value }) => key === 'span.kind' && value === 'client');
}
```
it's a matter of taste, I would prefer the first option.

Reply:
> it's a matter of taste

I know :-) Joe and Everett also use const a lot - I don't get it. When defining a function why not use the function syntax? Lambdas are harder to read. I guess you're saving on typing `return`.

Anyway, I just mostly wanted the return type to be declared.